presto-hive-metastore/src/main/java/com/facebook/presto/hive/HiveType.java

```diff
@@ -227,6 +227,9 @@ public static boolean isSupportedType(TypeInfo typeInfo)
     public static HiveType valueOf(String hiveTypeName)
     {
         requireNonNull(hiveTypeName, "hiveTypeName is null");
+        if (hiveTypeName.equals("uuid")) {
```

Comment:
```suggestion
        if (hiveTypeName.equals(HIVE_UUID.getTypeInfo().getTypeName())) {
```

Better to use `HIVE_UUID.getTypeInfo().getTypeName()`, since it is where the value `uuid` from.